CMakeLists.txt

```diff
@@ -7,7 +7,7 @@ set(CMAKE_POSITION_INDEPENDENT_CODE ON)
 set_property(CACHE CMAKE_BUILD_TYPE PROPERTY STRINGS "Debug" "RelWithDebInfo" "Release")
 if(NOT CMAKE_BUILD_TYPE)
   get_property(helpstring CACHE CMAKE_BUILD_TYPE PROPERTY HELPSTRING)
-  set(CMAKE_BUILD_TYPE RelWithDebInfo CACHE STRING ${helpstring} FORCE)
+  set(CMAKE_BUILD_TYPE Debug CACHE STRING ${helpstring} FORCE)
```

Comment:
Should remain RelWithDebInfo

Reply:
Needed a test run with LOGMSG_DEBUG to show up, but alas no crash. Put it back to RelWithxxx and converted loglevel to LOGMSG_ERROR instead.